db/seeds.rb

```diff
@@ -1,9 +1,9 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 #
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-role = Role.create!([{name: 'organizer'}])
-user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
\ No newline at end of file
+# role = Role.create!([{name: 'organizer'}])
```

Comment:
Could you remove these commented lines?

And the unneeded newlines (here and in the very last line of the file)